app/Http/Requests/Api/UpdateProfileRequest.php

```diff
@@ -24,7 +24,8 @@ public function rules()
     {
         return [
             'name' => 'required|string|max:50',
-            'address'        => 'string|max:255',
+            'password' => ($this->password == null) ? '' : 'required|min:6|max:50',
```

Comment:
em thêm cái `password ` vào `except` trong middleware `trim` là nó sẽ trim sẵn cho em rồi. k cần phải if chỗ này

Reply:
nhưng nếu bỏ `($this->password == null)` thì lúc edit profile em không nhập password thì bị lỗi requied sao anh

Reply:
em làm như trên a chỉ là đc, nó sẽ trim ở middleware trước khi vào request này